tools/xtask/config.toml

```diff
@@ -1,12 +1,22 @@
 licenses = ["LICENSE-MIT"]
 
-[rust_workspace_audit.ignore]
+# Global
+rust_workspace_audit.global_ignore = [
+    "syn",
+    "quote",
+    "proc-macro2",
+    "tauri-plugin-cli",
```

Comment:
Why is the `tauri-plugin-cli` plugin a global exception? It’s a desktop-specific exception and shouldn’t be used anywhere else.